app/grants/migrations/0005_grant_slug.py

```diff
@@ -4,13 +4,13 @@
 import django_extensions.db.fields
 
 
-def update_grants(apps, schema_editor):
-    """Handle the data migration for grant.slug."""
-    Grant = apps.get_model('grants', 'Grant')
-
-    grants = Grant.objects.all()
-    for grant in grants:
-        grant.save()
+# def update_grants(apps, schema_editor):
+#     """Handle the data migration for grant.slug."""
```

Comment:
did you mean to commit this?

Reply:
Yeah, probably dont need to comment this function